docs/source/io_formats/data_wmp.rst

```diff
@@ -0,0 +1,92 @@
+.. _io_data_wmp:
+
+==========================================
+The Windowed Multipole Library Format v0.2
```

Comment:
I'd make this "Windowed Multipole Library Format" and then have a sentence below saying this is version 0.2. More importantly, the library itself really should have the version information within it!